Simplified/Utilities/Authentication/NYPLBasicAuth.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  NYPLBasicAuth.swift
+//  Simplified
+//
+//  Created by Jacek Szyja on 02/07/2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+@objc class NYPLBasicAuth: NSObject {
+    typealias BasicAuthCompletionHandler = (URLSession.AuthChallengeDisposition, URLCredential?) -> Void
+
+    @objc static func authHandler(challenge: URLAuthenticationChallenge,
+                                  completionHandler: @escaping BasicAuthCompletionHandler)
+    {
+        switch challenge.protectionSpace.authenticationMethod {
+        case NSURLAuthenticationMethodHTTPBasic:
+            let account = NYPLUserAccount.sharedAccount()
+            if let barcode = account.barcode, let pin = account.PIN {
+                authCustomHandler(challenge: challenge,
+                                  completionHandler: completionHandler,
+                                  username: barcode,
+                                  password: pin)
+            } else {
+                completionHandler(.performDefaultHandling, nil)
```

Comment:
I think this was changed from `.cancel` to `. performDefaultHandling`, is this correct? just checking, .cancel seemed more intuitive at first glance but of course I'm probably wrong.

Reply:
actually, you're right